libraries/service_helpers.rb

```diff
@@ -0,0 +1,58 @@
+module CernerSplunk
+  # Mixin Helper methods for the Splunk Service resource
+  module ServiceHelpers
+    include ResourceHelpers
+    require 'win32/service' if RUBY_PLATFORM =~ /mswin|mingw|windows/
+
+    # Reports if the Windows service for the current package exists
+    # @return [Boolean]
+    def service_exists
+      ::Win32::Service.exists?(process_name)
+    end
+
+    def splunk_bin_path
+      Pathname.new(install_dir).join('bin')
+    end
+
+    # Returns the process id of the daemon is running
+    # @return [Integer]
```

Comment:
I might have been using ||= at one point, but apparently not now. I should be able to return nil.
Returning zero would be easier, but I want something falsey (and not interpretable as a valid number/pid)